src/templates/footer/footer.css

```diff
@@ -1,8 +1,24 @@
 .footer {
```

Comment:
Removed height and margin from CSS, used Bootstrap instead. Still need to look at the footer behavior because it needs to behave different on certain pages whose are a static height (of the screen).